store/src/main.rs

```diff
@@ -39,12 +40,12 @@ async fn main() -> Result<()> {
 /// Sends a gRPC request as specified by `command`.
 ///
 /// The request is sent to the endpoint defined in `config`.
+#[instrument(target = "miden-store", skip_all)]
 async fn query(
```

Comment:
Why do we need to instrument this? As I mentioned previously, this command will be run from the CLI - so, not sure we need to collect any logs here.